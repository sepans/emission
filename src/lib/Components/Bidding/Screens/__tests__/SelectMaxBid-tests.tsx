import React from "react"
import "react-native"
import * as renderer from "react-test-renderer"

import { SelectMaxBid } from "../SelectMaxBid"

it("renders properly", () => {
  const bg = renderer.create(<SelectMaxBid saleArtworkID="5aada729139b216c0bf18103" />).toJSON()
  expect(bg).toMatchSnapshot()
})
