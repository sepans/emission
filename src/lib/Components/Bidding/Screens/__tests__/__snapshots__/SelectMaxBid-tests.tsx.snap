// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<View
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "space-between",
        "marginBottom": 20,
        "marginLeft": 20,
        "marginRight": 20,
        "marginTop": 20,
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "AGaramondPro-Regular",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 10,
          "marginLeft": 10,
          "marginRight": 10,
          "marginTop": 10,
          "textAlign": "center",
        },
        undefined,
      ]
    }
  >
    Your max bid
  </Text>
  <View
    style={
      Array [
        Object {
          "width": "100%",
        },
        undefined,
      ]
    }
  >
    <RCTPicker
      items={
        Array [
          Object {
            "label": "$35,000 USD",
            "textColor": undefined,
            "value": 3500000,
          },
          Object {
            "label": "$40,000 USD",
            "textColor": undefined,
            "value": 4000000,
          },
          Object {
            "label": "$45,000 USD",
            "textColor": undefined,
            "value": 4500000,
          },
          Object {
            "label": "$50,000 USD",
            "textColor": undefined,
            "value": 5000000,
          },
          Object {
            "label": "$55,000 USD",
            "textColor": undefined,
            "value": 5500000,
          },
        ]
      }
      onChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      selectedIndex={2}
      style={
        Array [
          Object {
            "height": 216,
          },
          undefined,
        ]
      }
    />
  </View>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(51, 51, 51, 1)",
        "borderColor": "rgba(248, 248, 248, 1)",
        "borderWidth": 1,
        "height": 36,
        "justifyContent": "center",
      }
    }
    testID={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontSize": 12,
          },
          Object {
            "color": "rgba(204, 204, 204, 1)",
            "fontFamily": "AGaramondPro-Regular",
            "opacity": 1,
          },
          Object {
            "fontFamily": "AvantGardeGothicITC",
          },
        ]
      }
    >
      NEXT
    </Text>
  </View>
</View>
`;
